src/main/java/com/royalrangers/service/UserService.java

```diff
@@ -373,7 +380,7 @@ public void resendConfirmation(String email) throws UserRepositoryException, Unk
         if (!isEmailExist(email)) {
             throw new UserRepositoryException("User with this email is not exist.");
         }
-        if (user.getConfirmed()) {
+        if (!user.getConfirmed()) {
```

Comment:
I don't known.... I didn't make this change by hand, may be merging or conflict resolving mistake